KubeArmor/monitor/systemMonitor.go

```diff
@@ -393,7 +407,70 @@ func (mon *SystemMonitor) TraceSyscall() {
 	Containers := *(mon.Containers)
 	ContainersLock := *(mon.ContainersLock)
 
-	execLogMap := map[uint32]tp.Log{}
+	ctxReplayMap := map[SyscallContext]ContextReplay{}
+	ctxReplayMapLock := new(sync.RWMutex)
+	ReplayChannel := make(chan []byte, SyscallChannelSize)
+
+	go func() {
+		for {
+			dataRaw, valid := <-ReplayChannel
+			if !valid {
+				continue
+			}
+			dataBuff := bytes.NewBuffer(dataRaw)
+			ctx, err := readContextFromBuff(dataBuff)
+			if err != nil {
+				continue
```

Comment:
Is it ok to continue here? If yes, better to put a few ms sleep here so that this can yield to other routines.